docs/model-dev-guide/dtrain/dtrain-implement.rst

```diff
@@ -24,110 +25,117 @@ training, as described in :ref:`cluster-configuration`.
 <determined.TrialContext.get_per_slot_batch_size>` and :func:`context.get_global_batch_size()
 <determined.TrialContext.get_global_batch_size>`, respectively. If ``global_batch_size`` is not
 evenly divisible by ``slots_per_trial``, the remainder is dropped.
 
-If :ref:`slots_per_trial <exp-config-resources-slots-per-trial>` is greater than the number of slots
-on a single agent, Determined schedules it over multiple machines. When scheduling a multi-machine
-distributed training job, Determined requires that the job uses all of the slots (GPUs) on an agent.
-For example, in a cluster that consists of 8-GPU agents, an experiment with :ref:`slots_per_trial
-<exp-config-resources-slots-per-trial>` set to ``12`` is never scheduled and will wait indefinitely.
-The section on :ref:`Scheduling Behavior <dtrain-scheduling>` describes this in more detail.
+If the value of :ref:`slots_per_trial <exp-config-resources-slots-per-trial>` is greater than the
+number of slots available on a single agent, Determined schedules it over multiple machines. When
+scheduling a multi-machine distributed training job, Determined requires that all slots (GPUs) on an
+agent are used by the job. For example, in a cluster composed of 8-GPU agents, an experiment that
+has :ref:`slots_per_trial <exp-config-resources-slots-per-trial>` set to ``12`` will not be
+scheduled and will wait indefinitely. For more details, you can visit :ref:`Scheduling Behavior
+<dtrain-scheduling>`.
 
-There might also be running tasks preventing your multi-GPU trials from acquiring enough GPUs on a
-single machine. Consider adjusting ``slots_per_trial`` or terminating existing tasks to free slots
-in your cluster.
+You might have existing tasks that are running on a single machine that are preventing your
+multi-GPU trials from acquiring sufficient GPUs. To alleviate this, you may want to consider
+adjusting ``slots_per_trial`` or terminating existing tasks to free up slots in your cluster.
 
 Global Batch Size
 =================
 
-When doing distributed training, the ``global_batch_size`` specified in the
+During distributed training, the ``global_batch_size`` specified in the
 :ref:`experiment-config-reference` is partitioned across ``slots_per_trial`` GPUs. The per-GPU batch
 size is set to: ``global_batch_size // slots_per_trial``. If ``slots_per_trial`` does not divide
 ``global_batch_size`` evenly, the remainder is dropped. For convenience, the per-GPU batch size can
 be accessed via the Trial API, using :func:`context.get_per_slot_batch_size
 <determined.TrialContext.get_per_slot_batch_size>`.
 
-For improved performance, *weak-scaling* is recommended. That is, increasing your
-``global_batch_size`` proportionally with ``slots_per_trial``. For example, change
-``global_batch_size`` and ``slots_per_trial`` from 32 and 1 to 128 and 4.
+For improved performance, *weak-scaling* is recommended. Weak-scaling means proportionally
```

Comment:
nit: seems annoyingly technical to introduce jargon (weak-scaling) that is only used in this immediate sentence and without any link that could explain the concept further.

Reply:
If you think a paper link is appropriate, I would suggest [this one](https://arxiv.org/abs/1812.06162). But I was just suggesting removing the phrase `weak-scaling` completely.

If you think it's fine as it is, it can also be left alone. Just a personal preference thing.

Reply:
resolved via slack